scripts/update-konflux-refs.py

```
@@ -172,34 +225,26 @@ def main():
-                line = line.replace(current_image, new_image)
-
-            updated.append(line)
-
+        chunks = array_split(file_content, workers)
+        with ThreadPoolExecutor(max_workers=workers) as executor:
+            # Use future objects as a key to preserve submission order.
+            future_chunks = {executor.submit(process_chunk, chunk, max_tag_length): [] for chunk in chunks}
+            for future in as_completed(future_chunks):
+                try:
+                    future_chunks[future] = future.result()
+                except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)